test/toucan/db_test.clj

```diff
@@ -4,7 +4,8 @@
                     [models :as models])
             (toucan.test-models [user :refer [User]]
                                 [venue :refer [Venue]]
-                                [category :refer [Category]])
+                                [category :refer [Category]]
+                                [address :refer [Address]])
```

Comment:
Mind keeping these sorted?

Reply:
Oops. I just realized they're not sorted in the first place. Nevermind I'll sort these next time I push out a new release

Reply:
Done 😃